### ISSUE TITLE
Why are we using !important at this point?

### ISSUE THREAD

**Issue by joseegm** [SAP/techne#660]
At 
https://github.com/SAP/techne/blob/v2.0/dist/css/techne-all.css#L303
and successive style declarations. The `!important` declaration is present.

Overriding statements should be avoid at all cost, since they are hard to follow and debug. 
At this early point in the library we should not have to use `!important`.

Strive on the future to not use `!important` work with the language feature of CSS not against them.

**Comment by xak** [comment]
These classes are helpers and are intentionally added to the end of the master stylesheet. These should not be used very often, should not be relied upon but sometimes you need them. 

For example, I'm creating a page with full bleed bands of content down a page and they have different background colors. I could create a new class for each with the correct background color. But in this case a helper is appropriate instead of adding one-off classes to the global CSS.

The `!important` is here intentionally to override an other rules. If I had this markup ...

```
<div class="container">
	<div class="foo">
	</div>
	<div class="foo has-background-color-yellow">
<!-- this should have a yellow background to highlight it -->
	</div>
</div>
```

And this CSS ...

```
.container .foo {
   background-color: white;
}
.has-background-color-yellow {
  background-color: yellow !important;
}
```

For one-off cases, the helper is better than a new rule, such as `foo--highlighted`.

The `!important` flag is appropriate here. You will find no other `!important` flags anywhere else other than helpers. They are intended to provide brute-force when you need it but should be used sparingly.

No change is required. This implementation actually works specifically **with** the CSS cascade language features.

**Comment by joseegm** [comment]
As long as we stick with only using `!important` only for the helpers...
